### ISSUE TITLE
Add parks pacific marine fish codes 

### ISSUE THREAD

**Issue by aylapear** [poissonconsulting/fishbc#10]
## Your Idea

- Add the fish codes from the Parks Canada marine study to the package
- Add Lynn Lee as contributor

**Comment by aylapear** [comment]
I suggest adding the new table as `marineparks` 

@evanamiesgalonski & @joethorley What do you think the new table name should be?

**Comment by aylapear** [comment]
We assume the codes should be added to the `freshwaterfish` as a crossrefence (like the CDCode and ABCode columns)  for the ones that span both tables like salmon. Do you agree @evanamiesgalonski ?

**Comment by evanamiesgalonski** [comment]
i think `marineparksfish` would be a good name for the new table. And yes I agree it would be good to have an FK column in `freshwaterfish` to join to the new table.

**Comment by joethorley** [comment]
Yes agree with adding a FK to freshwater fish for where they match up.

**Comment by joethorley** [comment]
Our other tables are called ab and cdc so pc would be most fitting but they may have codes for freshwater - so how about `pcmarine`?

**Comment by Alcon98** [comment]
Oh I like the way that sounds, really rolls off the tongue
